source/systems/happinessandsaturation.md

```diff
@@ -6,19 +6,23 @@ layout: default
 
 The Saturation and Happiness systems for the citizens are locked in to each other, which means that the citizens require food to increase their happiness.
 
-The citizens, however, will NOT die if their saturation goes down to 0, they will simply stop leveling anymore and will also request food in chat regularly as well as have the slowness effect. The workers will stop working. This will affect the overall happiness of each citizen as well as the entire colony.
+The citizens, however, will NOT die if their saturation goes down to 0, they will simply stop leveling, request food in chat regularly, and gain the slowness effect. The workers will stop working. This will affect the overall happiness of each citizen as well as the entire colony.
```

Comment:
```suggestion
The citizens, however, will NOT die if their saturation goes down to 0. They will simply stop leveling, request food in chat regularly, and gain the slowness effect. The workers will stop working. This will affect the overall happiness of each citizen as well as the entire colony.
```